core/translations/pt.json

```diff
@@ -1655,7 +1695,12 @@
     "reset__finished_verifying_group_template": "Você concluiu a verificação dos seus shares de recuperação para o grupo {0}.",
     "reset__finished_verifying_shares": "Você concluiu a verificação dos seus shares de recuperação.",
     "reset__finished_verifying_wallet_backup": "Você concluiu a verificação do backup da sua carteira.",
-    "reset__group_description": "Um grupo é formado por shares de recuperação.",
+    "reset__group_description": {
```

Comment:
WDYT about shortening the title to fit in one line (e.g. `Definir nº de grupos`, as done by `reset__slip39_checklist_set_num_groups`)?

<img width="763" height="245" alt="image" src="https://github.com/user-attachments/assets/08f1a5df-d2f0-4db1-aa76-49b2c3a57cf7" />